workshops/filters.py

```diff
@@ -68,6 +68,10 @@ class EventFilter(django_filters.FilterSet):
     ]
     status = EventStateFilter(choices=STATUS_CHOICES)
 
+    invoice_status = django_filters.ChoiceFilter(
+        choices=((None, '---------'), ) + Event.INVOICED_CHOICES,
```

Comment:
The long dash now appears 3 times in filters.py - time to stick it in a constant at the top of the file?